### PR TITLE
Adjusted type names

### PR DESCRIPTION
**Pull request by suhasasumukh**
Adjusted type names to follow conventional PascalCase for interfaces and camelCase for properties.

### CODE REVIEW

**Comment by RevTpark** [comment]
Just done adding eslint and prettier, need to solve merge conflicts

File: types.ts [hunk 1]
```diff
@@ -22,45 +22,45 @@ export interface SkillsMapping {
...
+    skills: Skill[];
+    type: string;
+    requirements: any;
+    totalPaymentsMade: number;
+    totalWinnersSelected: number;
+    isWinnersAnnounced: number;
+    templateId: string;
+    region: string;
+    pocSocials: any;
+    hackathonPrize: number;
```
Comment (RevTpark) [inline types.ts:56]: This isn't something I named, this is how it appears in the data that comes through the query

**Review by RevTpark — CHANGES_REQUESTED** [review]

File: types.ts (hunk 1 above)
Comment (suhasasumukh) [reply types.ts]: fixed the name

**Comment by RevTpark** [comment]
1. There are still merge conflicts
2. Did you test this, the imports in index.ts are invalid

**Pull request closed without merging**